### PR TITLE
CLOUD-833: Provide docs for context-sensitive help in UI form

### PR DESCRIPTION
**Pull request by Bonrob2**
CLOUD-833: updated the text to standardize use of noun form instead of verb form, and to remove periods.

### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/CLOUD-833

### CODE REVIEW

**Review by bouff — APPROVED** [review]
LGTM 👍

File: opennms-container/minion/minion-config-schema.yml [hunk 1]
```diff
@@ -38,86 +38,86 @@ categories:
...
         label: "Secure Credentials Vault"
-        description: "Configuration for secure credentials vault (scv)."
+        description: "Configuration for secure credentials vault (SCV)"
         config:
           - name: "provider"
             type:
               name: "string"
             label: "Provider"
-            description: "Set the scv provider."
+            description: "Secure credentials valut (SCV) provider"
```
Comment (indigo423) [inline opennms-container/minion/minion-config-schema.yml:120]:
```suggestion
            description: "Secure credentials vault (SCV) provider"
```

File: opennms-container/minion/minion-config-schema.yml [hunk 2]
```diff
@@ -38,86 +38,86 @@ categories:
...
         label: "gRPC"
-        description: "Configure the Dominion gRPC endpoint."
+        description: "Configuration for the Dominion gRPC endpoint"
         config:
           - name: "host"
             type:
               name: "host"
             label: "Host"
-            description: "Set gRPC host for Dominion gRPC endpoint."
+            description: "gRPC host for Dominion gRPC endpoint"
```
Comment (indigo423) [inline opennms-container/minion/minion-config-schema.yml:94]:
```suggestion
            description: "Host for Dominion gRPC endpoint"
```

File: opennms-container/minion/minion-config-schema.yml [hunk 3]
```diff
@@ -38,86 +38,86 @@ categories:
...
             key: "/dominion/grpc/host"
             example: "my-dominion-grpc-host"
           - name: "port"
             type:
               name: "port"
             label: "Port"
-            description: "Set gRPC TCP port for Dominion gRPC endpoint."
+            description: "gRPC TCP port for Dominion gRPC endpoint"
```
Comment (indigo423) [inline opennms-container/minion/minion-config-schema.yml:101]:
```suggestion
            description: "TCP port for Dominion gRPC endpoint"
```

**Review by indigo423 — COMMENTED** [review]
Minor tweaks, LGTM! 👍

**Comment by Bonrob2** [comment]
Am going to resubmit this under different (cloned) Jira ticket and built from the foundation 2021 branch.

**Comment by Bonrob2** [comment]
Am going to resubmit this under different (cloned) Jira ticket (NMS-13255) and built from the foundation 2021 branch.

**Pull request closed without merging**